python/tvm/relax/transform/lazy_transform_params.py

```diff
@@ -149,9 +156,13 @@ def transform(self, func: relax.Function) -> relax.Function:
         # Step 3. rewrite get item and set item
         new_body = self.visit_expr(func.body)
 
-        # Step 4. Find all shape parameters that should be retained as
-        # parameters.
+        # Step 4. Add parameters of get_item and set_item (except index) to the function.
         params = []
+        for param in itertools.chain(self.get_item_param, self.set_item_param):
```

Comment:
Python's `*` expansion avoids the need for a loop.

```python
params = [*self.get_item_param, *self.set_item_param]
```

It's a bit more "magic", but avoids mutation of a variable after assignment.